src/pepr/uds-cluster-crds/Chart.yaml

```
@@ -0,0 +1,21 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+apiVersion: v2
+name: uds-cluster-crds
+description: UDS Core cluster level configurations
```

Description is probably more accurate as:
```suggestion
description: UDS Core CRDs
```
Since the "configurations" are really the CRs, not the CRDs?